### PR TITLE
add select2 for channels, sentry for production only, remove rubocop rules that break the linter

### PR DESCRIPTION
**Pull request by vikdotdev**
- [x] merged with the latest master branch
- [x] resolved all the conflicts, PR could be merged
- [x] new code has new specs
- [x] for new models - new factories were added too
- [x] unit specs are passed
- [x] code is well-formatted, Ruby, CSS, etc
- [x] screenshots of new pages, features are added
- [x] code passed internal team code review
- [x] no instance variables in partials
- [x] code is well formatted and has spaces!!!

PS: delete your branch in github after merge http://prntscr.com/q80mvq

### CODE REVIEW

**Comment by vikdotdev** [comment]
channel_content
2020-01-08T14:22:58
2020-01-08T14:23:11

File: db/migrate/20200107132719_add_contents_to_channels.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddContentsToChannels < ActiveRecord::Migration[6.0]
+  def change
+    add_column :channels, :contents, :integer, array: true, default: []
```
Comment (demchyna) [inline db/migrate/20200107132719_add_contents_to_channels.rb]: habtm

File: spec/models/channel_spec.rb [hunk 2]
```diff
@@ -23,4 +23,8 @@
       expect { channel.destroy }.to change { ActionMailer::Base.deliveries.count }.by(1)
     end
   end
+
+  it 'has contents' do
+    expect(channel.contents.count).to eq(3)
```
Comment (vikdotdev) [inline spec/models/channel_spec.rb]: тут напевно щось краще можна придумати, лиш не знаю що

File: spec/controllers/account/channels_controller_spec.rb [hunk 3]
```diff
@@ -130,6 +130,18 @@
...
         expect(channel.name).to eq('Danialberg')
       end
+
+      it 'has multiple contents' do
+        expect do
+          post :create, params: {
+            channel: {
+              name: 'Rommel',
+              organization_id: channel.organization.id,
+              contents: [create(:video), create(:page)]
```
Comment (romaslmd) [inline spec/controllers/account/channels_controller_spec.rb]: that's not good, are u really submit AR instances from the browser?

File: spec/features/channel_select_multiple_contents_spec.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+RSpec.feature 'Multiple contents select', type: :feature do
+  let!(:organization) { create(:organization) }
+  let!(:video) { create(:video, organization: organization) }
+  let!(:gallery) { create(:gallery, organization: organization) }
+  let!(:admin) { create(:admin, organization: organization) }
+
+  CHANNEL_NAME = 'select multiple example channel'
+  it 'creates channel with multiple contents, then updates it' do
+    visit '/'
+    click_link 'Sign In'
```
Comment (romaslmd) [inline spec/features/channel_select_multiple_contents_spec.rb]: stub sin in
u dont need to do it every time

File: spec/features/channel_select_multiple_contents_spec.rb [hunk 5]
```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.feature 'Multiple contents select', type: :feature do
+  let!(:organization) { create(:organization) }
+  let!(:video) { create(:video, organization: organization) }
+  let!(:gallery) { create(:gallery, organization: organization) }
+  let!(:admin) { create(:admin, organization: organization) }
+
+  CHANNEL_NAME = 'select multiple example channel'
```
Comment (romaslmd) [inline spec/features/channel_select_multiple_contents_spec.rb]: move to let, there is no need of constant

File: spec/features/channel_select_multiple_contents_spec.rb [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    fill_in 'Email', with: admin.email
+    fill_in 'Password', with: 'passwd'
+    click_button 'Login'
+    visit '/account/channels'
+
+    click_link 'Add New Channel'
+    fill_in 'Name', with: CHANNEL_NAME
+    select = page.find('.js-select2-contents')
+    select.find("option[value='#{video.id}']").select_option
+    select.find("option[value='#{gallery.id}']").select_option
```
Comment (romaslmd) [inline spec/features/channel_select_multiple_contents_spec.rb]: note: here u dont use select 2 selectors

File: spec/features/channel_select_multiple_contents_spec.rb [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    click_button 'Create Channel'
+    visit '/account/channels'
```
Comment (romaslmd) [inline spec/features/channel_select_multiple_contents_spec.rb]: separate logical groups to visually separate scenario and readability o

File: spec/controllers/account/channels_controller_spec.rb (hunk 3 above)
Comment (igorkasyanchuk) [reply spec/controllers/account/channels_controller_spec.rb]: + це буде контент з якої організації ?

де ти сабмітаєш ІД ?

де перевірка що КАНАЛ МАЄ 2 контента зв"язані? тут тест в тебе не правильний!!!

File: spec/features/channel_select_multiple_contents_spec.rb (hunk 4 above)
Comment (igorkasyanchuk) [reply spec/features/channel_select_multiple_contents_spec.rb]: та вже хай буде :) хай більше фіч роблять

File: spec/features/channel_select_multiple_contents_spec.rb (hunk 7 above)
Comment (igorkasyanchuk) [reply spec/features/channel_select_multiple_contents_spec.rb]: то я сказав так зробити, повний сценарій юзера

File: spec/features/channel_select_multiple_contents_spec.rb (hunk 6 above)
Comment (demchyna) [reply spec/features/channel_select_multiple_contents_spec.rb]: OK for now (Igor)

**Pull request merged**